reconcile/utils/terrascript_aws_client.py

```diff
@@ -3994,6 +3996,51 @@ def populate_tf_resource_elasticsearch(self, spec):
             "internal_user_database_enabled", False
         ):
             master_user = security_options["master_user_options"]
+            secret_values = {"name": identifier}
```

Comment:
I assume the `name` must be unique across the region? Would it make sense to add a prefix for the integration to avoid potential collisions with future identifiers from other integrations? 

```python
secret_values = {"name": f"qr-es_{identifier}"}
```